todoman/cli.py

```diff
@@ -48,6 +48,23 @@ def _validate_date_param(ctx, param, val):
         raise click.BadParameter(e)
 
 
+def _validate_start_date_param(ctx, param, val):
+    try:
+        if 'before' in val:
+            temp = val[7:]
+            ret = [True]
+        elif 'after' in val:
+            temp = val[6:]
+            ret = [False]
+        else:
+            raise click.BadParameter(
+                'The start date of the task should be in format \'before 2012-10-16\'')
+        ret.append(ctx.obj['formatter'].parse_date(temp))
+        return ret 
```

Comment:
Instead of a list I'd prefer a tuple. Because tuples are immutable you'll have to create it in one step. Something like this:

    if ..:
        is_before = True
    elif ..:
        is_before = False
    else:
        ..
    return is_before, ctx.obj['formatter'].parse_datetime(temp)

Also please use a better variable name than `temp`.